File: simpleio/src/main/java/ch/bailu/simpleio/parser/OnParsedInterface.java
package ch.bailu.simpleio.parser;

import java.io.IOException;


public interface OnParsedInterface {
    public void onHaveSegment();
    public void onHavePoint() throws IOException;
    
    
    public final static OnParsedInterface NULL = new OnParsedInterface() {
		@Override
		public void onHaveSegment() {}
		@Override
		public void onHavePoint() {}
	};
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/xml/parser/util/OnParsedInterface.kt
package ch.bailu.aat_lib.xml.parser.util;

import java.io.IOException;


public interface OnParsedInterface {
    void onHaveSegment();
    void onHavePoint() throws IOException;
    
    
    OnParsedInterface NULL = new OnParsedInterface() {
		@Override
		public void onHaveSegment() {}
		@Override
		public void onHavePoint() {}
	};
}
